### PR TITLE
Javascript javascript2 week2

### PR DESCRIPTION
**Pull request by VioMNE**

### CODE REVIEW

File: javascript/javascript2/week2/main.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
+console.log("Script loaded");
+
+const products = getAvailableProducts();
+console.log(products);
```
Comment (KavyaReddy732) [inline javascript/javascript2/week2/main.js:4]: Try not to include console.log in PR's

File: javascript/javascript2/week2/main.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+      if (a.name < b.name) {
+        return -1;
+      } else if (a.name > b.name) {
+        return 1;
+      } else {
+        return 0;
+      }
+    });
+  }
+}
```
Comment (KavyaReddy732) [inline javascript/javascript2/week2/main.js:74]: very nice 👍

**Review by KavyaReddy732 — COMMENTED** [review]
very neat code and well done 👍.

**Comment by KavyaReddy732** [comment]
for movies.js i can't open the file in PR. try to exclude the array of movies and upload just code you have written. so it'll be shown in the PR.

File: javascript/javascript2/week2/main.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+  } else if (sort.value === "rating") {
+    sortBy.sort((a, b) => a.rating - b.rating);
+  } else if (sort.value === "name") {
+    sortBy.sort((a, b) => {
+      if (a.name < b.name) {
+        return -1;
+      } else if (a.name > b.name) {
+        return 1;
+      } else {
+        return 0;
```
Comment (piepear) [inline javascript/javascript2/week2/main.js:70]: Try to write the return value to be more readable for the code. Also remember that when a code goes into the return line, it won't execute the rest of the function, so you wouldn't need the last else statement, and can just return 0.

**Review by Yuliia-Balandiuk — COMMENTED** [review]
Hi Violeta.  Good job!
 I also can't open movies.js file.  So maybe you can add code with movie exercise  to new one.

**Pull request merged**